source/manual/data-gov-uk-common-tasks.html.md

```diff
@@ -1,240 +0,0 @@
----
-owner_slack: "#datagovuk-tech"
-title: Common tasks for data.gov.uk
-section: data.gov.uk
-layout: manual_layout
-parent: "/manual.html"
-last_reviewed_on: 2018-04-30
-review_in: 3 months
----
-[publish]: apps/datagovuk_find
-[find]: apps/datagovuk_find
-[paas]: https://docs.cloud.service.gov.uk/#technical-documentation-for-gov-uk-paas
-[paas-backup]: #populate-the-postgres-and-elasticsearch-services-on-paas
-[infrastructure]: https://github.com/alphagov/datagovuk_infrastructure
-
-## Deploy Find Data and Publish Data to production
```

Comment:
Is this being moved somewhere else?  I think it would be handy to keep somewhere are we don't deploy these apps in the standard GOV.UK way.  Perhaps in the apps' README?

Reply:
There's now a 'deployment' manual, which covers this sort of thing. I'm trying to make the ops manual about operational tasks to maintain the apps.

Reply:
👍